File: Examples/VisualBasic/ProgrammersGuide/WorkingWithProjects/WriteCurrencyProperties.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Tasks

Namespace Aspose.Tasks.Examples.VisualBasic.WorkingWithProjects
    Public Class WriteCurrencyProperties
        Public Shared Sub Run()
            'Exstart: WriteCurrencyProperties
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Projects()

            'Create a project instance
            Dim project As New Project(dataDir & "WriteCurrencyProperties.mpp")

            'Set currency properties
            project.Set(Prj.CurrencyCode, "AUD")
            project.Set(Prj.CurrencyDigits, 2)
            project.Set(Prj.CurrencySymbol, "$")
            project.Set(Prj.CurrencySymbolPosition, CurrencySymbolPositionType.After)

            'Save the project as XML project file
            project.Save(dataDir & "project.xml", Aspose.Tasks.Saving.SaveFileFormat.XML)
            'ExEnd: WriteCurrencyProperties
        End Sub
    End Class
End Namespace

File: Examples/CSharp/HandlingBasicProperties/WriteCurrencyProperties.cs
using Aspose.Tasks.Saving;

namespace Aspose.Tasks.Examples.CSharp.HandlingBasicProperties
{
    public class WriteCurrencyProperties
    {
        public static void Run()
        {
            // ExStart:WriteCurrencyProperties
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_HandlingBasicProperties();

            // Create a project instance
            Project project = new Project(dataDir+ "WriteCurrencyProperties.mpp");

            // Set currency properties
            project.Set(Prj.CurrencyCode, "AUD");
            project.Set(Prj.CurrencyDigits, 2);
            project.Set(Prj.CurrencySymbol, "$");
            project.Set(Prj.CurrencySymbolPosition, CurrencySymbolPositionType.After);

            // Save the project as XML project file
            project.Save(dataDir+ "project_out.xml", SaveFileFormat.XML);
            // ExEnd:WriteCurrencyProperties
        }
    }
}